src/Tasks/GetReferenceAssemblyPaths.cs

```diff
@@ -246,7 +246,14 @@ private IList<String> GetPaths(string rootPath, string targetFrameworkFallbackSe
                 // 1/26/16: Note this was changed from a warning to an error (see GitHub #173).
                 if (pathsToReturn.Count == 0)
                 {
-                    Log.LogErrorWithCodeFromResources("GetReferenceAssemblyPaths.NoReferenceAssemblyDirectoryFound", frameworkmoniker.ToString());
+                    if (frameworkmoniker.Identifier == "net" && frameworkmoniker.Version >= new Version(5, 0))
```

Comment:
Why allocate a new version?

Reply:
A combination of wanting to get the PR out before I signed off, and forgetting you could simply check `frameworkmoniker.Version.Major` 🙂